README.md

```diff
@@ -4,39 +4,39 @@
 Postfacto helps teams run great [retrospecives](https://content.pivotal.io/blog/how-to-run-a-really-good-retrospective) remotely.
```

Comment:
"Retroscives" -> "Retrospectives"

Reply:
😊 My Bad

Reply:
I went ahead and modified this in master given this has been pending a couple of days.